docs/scos/dev/feature-integration-guides/202204.0/mailing-and-notifications-feature-integration.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Mailing and notifications feature integration
+description: The guide describes the process of installing the SymfonyMailer in your project.
+last_updated: Nov 21, 2022
+template: feature-integration-guide-template
+---
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place. This guide only describes installing
+SymfonyMailer in order to provide mail sending via `symfony/mailer` component.
+
+{% endinfo_block %}
+
+## Install feature core
+
+Follow the steps below to install the feature core.
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+|---|---|---|
+| Spryker Core | {{site.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html)
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker/symfony-mailer:^1.0.0 --update-with-dependencies
```

Comment:
Should this be 1.0.1 as per https://release.spryker.com/release-groups/view/4395 include statement?
Since the patch was necessary to complete this release.